packages/flutter/test/widgets/sliver_prototype_item_extent_test.dart

```diff
@@ -136,4 +137,18 @@ void main() {
     for (int i = 1; i < 10; i += 1)
       expect(find.text('Item $i'), findsOneWidget);
   });
+
+  testWidgets('SliverPrototypeExtentList prototypeItem paint transform is zero.', (WidgetTester tester) async {
+    // Regression test for https://github.com/flutter/flutter/issues/67117.
```

Comment:
hint: Remove the trailing ".", since otherwise GitHub's linkifier adds it to the URL.